### PR TITLE
feat(autonomous_emergency_braking): add some tests to aeb

### PR DESCRIPTION
**Pull request by danielsanchezaran**
## Description
This PR adds several tests to the AEB module. It does not cover every possible function but aims to get a 50% coverage.

image

## Related links

**Parent Issue:**

- Link

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

File: launch/tier4_control_launch/launch/control.launch.xml [hunk 1]
```diff
@@ -192,7 +192,6 @@
         <remap from="~/input/pointcloud" to="/perception/obstacle_segmentation/pointcloud"/>
         <remap from="~/input/velocity" to="/vehicle/status/velocity_status"/>
         <remap from="~/input/imu" to="/sensing/imu/imu_data"/>
-        <remap from="~/input/odometry" to="/localization/kinematic_state"/>
```
Comment (kyoichi-sugahara) [inline launch/tier4_control_launch/launch/control.launch.xml:195]: @danielsanchezaran
This change is intended?

**Comment by kyoichi-sugahara** [comment]
@danielsanchezaran 
Implementation itself looks to me :+1: Thanks.
Could you also share the coverage after implemeting those test?
I expect following info and if you don't know how to check those info let me know.
image

File: launch/tier4_control_launch/launch/control.launch.xml (hunk 1 above)
Comment (danielsanchezaran) [reply launch/tier4_control_launch/launch/control.launch.xml:195]: @kyoichi-sugahara yes, that topic is not used

**Comment by danielsanchezaran** [comment]
> @danielsanchezaran Implementation itself looks to me 👍 Thanks. Could you also share the coverage after implemeting those test? I expect following info and if you don't know how to check those info let me know. image

Sure, 

image

**Review by kyoichi-sugahara — APPROVED** [review]
Thank you LGTM!

**Review by takayuki5168 — APPROVED** [review]

**Pull request merged**